scripts/workers/submission_worker.py

```diff
@@ -59,7 +58,7 @@
 SUBMISSION_DATA_DIR = join(SUBMISSION_DATA_BASE_DIR, 'submission_{submission_id}')
 SUBMISSION_INPUT_FILE_PATH = join(SUBMISSION_DATA_DIR, '{input_file}')
 CHALLENGE_IMPORT_STRING = 'challenge_data.challenge_{challenge_id}'
-
+RABBITMQ_PARAMETERS = settings.RABBITMQ_PARAMETERS
```

Comment:
I am actually thinking here is it required ? We can simply refer `settings.RABBITMQ_PARAMETERS`